plugins/source/aws/resources/services/lambda/functions.go

```diff
@@ -153,9 +152,7 @@ func resolveCodeSigningConfig(ctx context.Context, meta schema.ClientMeta, resou
 	svc := cl.Services(client.AWSServiceLambda).Lambda
 
 	// skip getting CodeSigningConfig since containerized lambda functions does not support this feature
-	// value can be nil if the caller doesn't have GetFunctionConfiguration permission and only has List*
-	lambdaType := resource.Get("code_repository_type").(*scalar.String)
```

Comment:
Not entirely sure why we previously relied on untyped `resource.Get` instead of directly accessing values from `r.Configuration`. Either way, this approach seemed to not be working for us as we were still getting errors from `GetRuntimeManagementConfig`.